README.md

```diff
@@ -1,57 +1,364 @@
+1. In the **Client secrets** section, click on **New client secret**:
+   - Type a key description (for instance `app secret`),
+   - Select one of the available key durations (**In 1 year**, **In 2 years**, or **Never Expires**) as per your security concerns.
+   - The generated key value will be displayed when you click the **Add** button. Copy the generated value for use in the steps later.
+   - You'll need this key later in your code's configuration files. This key value will not be displayed again, and is not retrievable by any other means, so make sure to note it from the Azure portal before navigating to any other screen or blade.
+
+1. In the app's registration screen, click on the **API permissions** blade in the left to open the page where we add access to the Apis that your application needs.
+   - Click the **Add permissions** button and then,
+   - Ensure that the **Microsoft APIs** tab is selected.
+   - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**
+   - In the **Delegated permissions** section, select the **User.ReadBasic.All** in the list. Use the search box if necessary.
+   - Click on the **Add permissions** button in the bottom.
+   - Click the check-marked button **Grant Admin Consent** to grant admin consent and confirm by clicking the `Yes` option in the pop up box.
+
+### Configure the web app (python-django-webapp-call-graph) to use your app registration
+
+Open the project in your IDE to configure the code.
+
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `aad.config.json` file
+1. Find the string `{enter-your-tenant-id-here}` and replace the existing value with your Azure AD tenant ID.
+1. Find the string `{enter-your-client-id-here}` and replace the existing value with the application ID (clientId) of the `python-django-webapp-call-graph` application copied from the Azure portal.
+1. Find the string `{enter-your-client-secret-here}` and replace the existing value with the key you saved during the creation of the `python-django-webapp-call-graph` app, in the Azure portal.
+
+</details>
+
+## Running the sample
+
+- To run the sample, open a terminal window. Navigate to the root of the project. Be sure your virtual environment with dependencies is activated ([Prerequisites](#prerequisites)).
+- On Linux/OSX via the terminal:
+
+  ```Shell
+    # start from the folder in which the sample is cloned into
+    python manage.py collectstatic
+    python manage.py migrate
+    python manage.py runserver localhost:8000
+  ```
+
+- On Windows:
+
+  ```PowerShell
+    # start from the folder in which the sample is cloned into
+    python manage.py collectstatic
+    python manage.py migrate
+    python manage.py runserver localhost:8000
+  ```
+
+- Navigate to [http://localhost:8000](http://localhost:8000) in your browser (**Don't use** use 127.0.0.1)
+
+![Experience](./ReadmeFiles/app.png)
```

Comment:
You need to talk more, even if it sounds trivial
1. Where to sign in and which account to use
2. then a line ablout the consent screen
3. ID token details and Call MS Graph API buttons
4. Sign-out and any other experiences like 401s